### PR TITLE
Cumulative patches to make xCAT run on RHEL 8 

### PR DESCRIPTION
**Pull request by neo954**
### This pull request includes patches to make xCAT run on RHEL 8

_##Feature description##_

### The content of the PR:
* [x] _##Tweak rpm package dependencies_
* [x] _##Remove support of RHEL 8 Partner Inspection Snapshot 1 and Partner Inspection Snapshot 2_
* [x] _##Porting patches of the `httpport' feature support_
* [x] _##Basic service node template and pkglist_
* [x] _##Use shell code to do the base64 decode instead of use python._
  On RHEL 8, only `python3` is provided. I cannot maintain the same code across RHEL 7 and RHEL 8. Thus, changed to shell code, just like what we did in Ubuntu.

### CODE REVIEW

File: xCAT/xCAT.spec [hunk 1]
```diff
@@ -56,7 +56,7 @@ Requires: /usr/sbin/dhcpd
 # On RHEL this pulls in openssh-server, on SLES it pulls in openssh
 Requires: /usr/bin/ssh
 %if %nots390x
-Requires: /etc/xinetd.d/tftp
+Requires: /usr/sbin/in.tftpd
```
Comment (robin2008) [inline xCAT/xCAT.spec]: Does this change work for SLES?

File: xCAT/postscripts/xcatpostinit1.install [hunk 2]
```diff
@@ -37,20 +37,20 @@ start)
   fi
   # if the xcatdsklspost file exists and this is a reboot - run xcatdsklspost with a mode of 6
   if [ "$REBOOT" = "TRUE" -a -r /opt/xcat/xcatdsklspost ] && [[ "$RUNBOOTSCRIPTS" =~ ^(1|yes|y)$ ]]; then
```
Comment (robin2008) [inline xCAT/postscripts/xcatpostinit1.install]: It seems all changes in this file are format, why we need to include in this PR, it make confusing.

File: xCAT/xCAT.spec (hunk 1 above)
Comment (neo954) [reply xCAT/xCAT.spec]: Yes, this change works for SLES.

File: xCAT/postscripts/xcatpostinit1.install (hunk 2 above)
Comment (neo954) [reply xCAT/postscripts/xcatpostinit1.install]: This commit will be removed.

**Comment by neo954** [comment]
Please merge https://github.com/xcat2/xcat-core/pull/5929 first. There are patches I have to do the code merge manually. Thus I have to merge the `master` branch frequently.

**Comment by robin2008** [comment]
Merge it to branch first.

**Comment by neo954** [comment]
Thanks @robin2008

**Pull request merged**